### PR TITLE
Reverse ETL source primary key setting

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## What do these changes do?

> Added the **Primary Key** dashboard setting used for JSON mapping

## Type of documentation

- [ ] New documentation
- [x] Documentation update
- [ ] Hotfix

### CODE REVIEW

File: docs/sources/reverse-etl/common-settings/importing-data-using-tables.mdx [hunk 1]
```diff
@@ -4,42 +4,46 @@ description: >-
...
 - **Sync mode**: Select the sync mode that RudderStack uses to sync your data.
 
 <div class="infoBlock">
-
 RudderStack supports two modes for syncing your data - <strong>Upsert</strong> and <strong>Mirror</strong>. For more information on these modes, refer to the <Link to="/sources/reverse-etl/common-settings/sync-modes/">Sync Modes</Link> guide.
 </div>
 
-- **Choose user identifier**: Choose atleast one user identifier from `user_id` and `anonymous_id` from the dropdown.
-- You will also be able to preview a snippet of your data, as shown:
+- **Primary Key**: Select the column to uniquely identify your user records.
```
Comment (vyeshwanth) [inline docs/sources/reverse-etl/common-settings/importing-data-using-tables.mdx:32]:
```suggestion
- **Primary Key**: Select the column to uniquely identify your records in warehouse.
```

Also a hint like, syncs use this primary key for incremental syncs

**Review by vyeshwanth — APPROVED** [review]
LGTM

File: docs/sources/reverse-etl/common-settings/importing-data-using-tables.mdx [hunk 2]
```diff
@@ -4,42 +4,46 @@ description: >-
...
 
-This guide lists the settings required to import and sync data from your warehouse table to the specified destination.
-
 <div class="infoBlock">
-
-This guide is applicable for Reverse ETL sources configured using the <strong>Table</strong> option. If you have configured your Reverse ETL source using the <strong>Model</strong> option, refer to the <Link to="/sources/reverse-etl/common-settings/importing-data-using-models/">Importing Data using Models</Link> guide.
+This guide is applicable for the Reverse ETL sources configured using the <strong>Table</strong> option. If you have configured your Reverse ETL source using the <strong>Model</strong> option, refer to the <Link to="/sources/reverse-etl/common-settings/importing-data-using-models/">Importing Data using Models</Link> guide.
 </div>
 
+RudderStack gives you two options to map your columns to specific destination fields while importing the data:
```
Comment (ishachopra0401) [inline docs/sources/reverse-etl/common-settings/importing-data-using-tables.mdx:11]:
```suggestion
RudderStack gives you two options to map your warehouse columns to specific destination fields while importing the data:
```

File: docs/sources/reverse-etl/common-settings/importing-data-using-tables.mdx [hunk 3]
```diff
@@ -4,42 +4,46 @@ description: >-
...
+
+- Map with <Link to="/sources/reverse-etl/features/visual-data-mapper/">Visualizer</Link> (Currently supported only for selective destinations)
```
Comment (ishachopra0401) [inline docs/sources/reverse-etl/common-settings/importing-data-using-tables.mdx:13]:
```suggestion
- Map with Visualizer (Refer to the <Link to="/sources/reverse-etl/features/visual-data-mapper/">Visualizer</Link> guide which mentions the supported destinations.)
```

File: docs/sources/reverse-etl/common-settings/importing-data-using-tables.mdx [hunk 4]
```diff
@@ -4,42 +4,46 @@ description: >-
...
 - **Sync mode**: Select the sync mode that RudderStack uses to sync your data.
 
 <div class="infoBlock">
-
 RudderStack supports two modes for syncing your data - <strong>Upsert</strong> and <strong>Mirror</strong>. For more information on these modes, refer to the <Link to="/sources/reverse-etl/common-settings/sync-modes/">Sync Modes</Link> guide.
 </div>
 
-- **Choose user identifier**: Choose atleast one user identifier from `user_id` and `anonymous_id` from the dropdown.
-- You will also be able to preview a snippet of your data, as shown:
+- **Primary Key**: Select the column to uniquely identify your records in the warehouse.
```
Comment (ishachopra0401) [inline docs/sources/reverse-etl/common-settings/importing-data-using-tables.mdx:32]:
```suggestion
- **Primary Key**: Select a column from the table specified above to uniquely identify your records in the warehouse.
```

**Review by ishachopra0401 — APPROVED** [review]

**Pull request merged**